doc/input-output-reference/src/overview/group-simulation-parameters.tex

```diff
@@ -690,6 +690,14 @@ \subsubsection{Inputs}\label{inputs-14-014}
 
 This field is used to alter the effective generic contaminant capacitance of the zone air volume. This affects the transient calculations of zone air generic contaminantconcentration. Values greater than 1.0 have the effect of smoothing or damping the rate of change in the generic contaminantlevel of zone air from timestep to timestep.
 
+\paragraph{Field: Name}\label{field-name-hybrid-model}
+
+The name of the ZoneCapacitanceMultiplier:ResearchSpecial object.
+
+\paragraph{Field: Zone or ZoneList Name}\label{field-Zone-or-zonelist-name-hybrid-model}
+
+This field is the name of the thermal zone (ref: Zone) and attaches a particular zone capacitance multiplier to a thermal zone or set of thermal zones in the building. When the ZoneList option is used then capacity multiplier is applied to each of the zones in the zone list.
```

Comment:
If you could actually put in a LaTeX `\ref` to the Zone definition, that would be much more helpful than just a text ref.